src/helpers/userAnalyticHelpers.js

```diff
@@ -6,17 +6,20 @@ export function segmentUA(data) {
   var analytics = (window.analytics = window.analytics);
 
   // Ensure that any UTM query parameters are sent along to Segment
-  var queryParameters = retrieveUTMQueryParameters();
-  var combinedData = Object.assign({}, data, queryParameters);
+  // var queryParameters = retrieveUTMQueryParameters();
+  // var combinedData = Object.assign({}, data, queryParameters);
+  var combinedData = stringifiedData;
```

Comment:
FYI the develop branch was not using stringifiedData before - it was using the data object/dict. This will probably behave differently.